deployment/node-feature-discovery/values.yaml

```diff
@@ -0,0 +1,195 @@
+image:
+  repository: gcr.io/k8s-staging-nfd/node-feature-discovery
```

Comment:
I think we should add image tag as a value, with `master` as default

Reply:
I think this has been addressed. Resolving the conversation